File: C#/BluffinMuffin.Protocol/Game/PlayerPlayMoneyCommand.cs
using BluffinMuffin.Protocol.DataTypes.Attributes;

namespace BluffinMuffin.Protocol.Game
{
    /// <summary>
    /// The client is telling the server that he plays some money.
    /// </summary>
    public class PlayerPlayMoneyCommand : AbstractGameCommand
    {
        /// <summary>
        /// Amount of money played
        /// </summary>
        [ExampleValue(42)]
        public int AmountPlayed { get; set; }
    }
}

File: Python/protocol/game/player_play_money_command.py
from protocol import AbstractGameCommand


class PlayerPlayMoneyCommand(AbstractGameCommand):
    def __init__(self, obj):
        super().__init__(obj)
        self.amount_played = obj['AmountPlayed']

    def __str__(self):
        return '{0} ({1})'.format(
            super().__str__(),
            self.amount_played
        )

    def _encode_specific(self, d):
        super()._encode_specific(d)
        d['AmountPlayed'] = self.amount_played
